sktime/forecasting/trend/_polynomial_trend_forecaster.py

```diff
@@ -126,6 +150,15 @@ def _fit(self, y, X, fh):
 
         # fit regressor
         self.regressor_.fit(X_sklearn, y)
+
+        if self.prediction_intervals:
+            # calculate and save values needed for the prediction interval method
+            fitted_values = self.regressor_.predict(X_sklearn)
+            residuals = y - fitted_values
+            p = self.degree + int(self.get_params()["with_intercept"])
```

Comment:
```suggestion
            p = self.degree + int(self.with_intercept)
```

Not necessarily suggesting a change, can you please confirm if this is same?

Reply:
yes, it must be, by the conventions of the API specifcation